File: С++/Example9.cpp
#include <iostream>

using namespace std;

int main() {
	int k = 1;
	// Simple switch
	// Instead K can be bool, int or char types
	switch (k) {
		case 0:
			cout << "zero" << endl;
			break;
		case 1:
			cout << "one" << endl;
			break;
		case 2:
			cout << "two" << endl;
			break;
		default:
			cout << "What is it?" << endl;
	}

	return 0;
}

File: Java/Example9.java
// We use this import, because better to write out.println() instead System.out.println();
import static java.lang.System.out;

public class Example9 {
	public static void main(String[] args) {
		int i = 0;
		// Simple switch
		switch (i) {
		// If i == 0
			case 0:
				out.println("Zero");
				break;
			// If i == 1
			case 1:
				out.println("One");
				break;
			// If i == 2
			case 2:
				out.println("Two");
				break;
			// Other variants
			default:
				out.println("Unknown value");
		}
	}
}
